journal/202105.md

```diff
@@ -0,0 +1,475 @@
+- show rich text [formatting](https://github.com/planetdecred/dcrandroid/pull/557) for Politeia proposals (without using WebView)
+- updated Chinese and French translations
+- fixed [watch-only](https://github.com/planetdecred/dcrandroid/pull/560) wallet being listed as a source when sending funds
+
+Merged in [dcrlibwallet](https://github.com/planetdecred/dcrlibwallet) shared library:
+
+- added code for fetching proposal [description](https://github.com/planetdecred/dcrlibwallet/pull/192) and switched from saving proposal files to loading them on demand
+- integrated a signed TLS [certificate](https://github.com/planetdecred/dcrlibwallet/pull/193) for [cspp.decred.org](https://cspp.decred.org/) which is required when using mainnet
+- fixed bugs with balance and Politeia integration
+
+<a id="dcrios" />
+
+**[dcrios](https://github.com/planetdecred/dcrios)**
+
+- implemented [Privacy mode](https://github.com/planetdecred/dcrios/pull/727) with guided setup of the mixer
+- show [USD equivalent](https://github.com/planetdecred/dcrios/pull/746) of wallet's DCR balance on the Overview page
+- added password input [delays](https://github.com/planetdecred/dcrios/pull/749) to resist brute force attacks
+- on the Overview page, show [which wallet](https://github.com/planetdecred/dcrios/pull/776) transactions belong to when there is more than one wallet
+- show more specific [notifications](https://github.com/planetdecred/dcrios/pull/766) for staking transactions (ticket voted or revoked)
+- show rich text [formatting](https://github.com/planetdecred/dcrios/pull/773) for Politeia proposals (without using UIWebView)
+- fixed incorrect list of [accounts](https://github.com/planetdecred/dcrios/pull/765) to send from
+- added support for iOS 11
+- ~17 bug fixes and UI tweaks
+
+<a id="godcr" />
+
+**[godcr](https://github.com/planetdecred/godcr)**
+
+- implemented pages: [Tickets](https://github.com/planetdecred/godcr/pull/395) overview, [Tickets list](https://github.com/planetdecred/godcr/pull/406), and [Restore wallet](https://github.com/planetdecred/godcr/pull/386)
+- implemented the [Max](https://github.com/planetdecred/godcr/pull/396) button for setting the maximum DCR amount that can be sent
+- added the submission of input fields with pressing [Enter](https://github.com/planetdecred/godcr/pull/414)
+- added [loading](https://github.com/planetdecred/godcr/pull/400) animations to certain modals
+- initial [dark mode](https://github.com/planetdecred/godcr/pull/401) suppot
+- added language translation infrastructure
+- ~9 bug fixes
+
+Elias Naur (creator of the Gio library that powers GoDCR) has kindly reviewed the UI code and shared his [recommendations](https://paste.sr.ht/~eliasnaur/cea1d29d6a5f96668b5e166c2f39ef596974574f), many of which have already been implemented. { can we link this publicly? }
+
+{ what is the "product name" for use in texts, "godcr" or "GoDCR"? }
+
+{ maybe GoDCR shot, 768px wide if possible }
```

Comment:
<img width="768" alt="Screenshot 2021-06-02 at 00 07 31" src="https://user-images.githubusercontent.com/15804688/120400815-8ca1db00-c336-11eb-9469-bf0d84baa3f8.png">

Reply:
Thanks for sharing the image! I'll add it with the other images at a later stage of the release cycle.